src/Hedgehog/Range.fs

```diff
@@ -54,20 +58,62 @@ module Range =
     //
 
     /// Construct a range which represents a constant single value.
+    ///
+    /// >>> Range.bounds x <| Range.singleton 5
+    /// (5, 5)
+    ///
+    /// >>> Range.origin <| Range.singleton 5
+    /// 5
```

Comment:
But it could be a good idea to integrate Doctest (when it reaches version 1) to the F# Formatting tools (which are somehow similar to haddock) 😉

I'm not that worried right now because there's no equivalent of hackage in .NET. There's actually the NuGet Gallery but it's just for searching the packages (and for that, I think Google does a much better job)...